deluge/tests/test_core.py

```diff
@@ -116,7 +117,13 @@ def on_shutdown(result):
     def add_torrent(self, filename, paused=False):
         if not paused:
             # Patch libtorrent flags starting torrents paused
-            self.patch(deluge.core.torrentmanager, 'LT_DEFAULT_ADD_TORRENT_FLAGS', 592)
+            self.patch(
+                deluge.core.torrentmanager,
+                'LT_DEFAULT_ADD_TORRENT_FLAGS',
+                lt.torrent_flags.auto_managed
+                | lt.torrent_flags.update_subscribe
+                | lt.torrent_flags.apply_ip_filter,
```

Comment:
I'm sure that using the value is intentional to avoid relying on `libtorrent` in a test. A comment would suffice or create variables with those values.

Reply:
The problem with this hard-coded value is the change of values in lt-1.2.
For testing this properly, without relying on `libtorrent`, we should set some environment flag to know which version of it we are using and set the proper value accordingly.

Do you have any suggestions for how to do this?

Reply:
Ah ok, then import and use the old-style lt flags.